src/app/cats/cats-detail-view/cats-detail-view.component.html

```diff
@@ -0,0 +1,36 @@
+<section>
+    <div class="side-by-side">
+      <button (click) ="scrollLeft()" class="scrolls"><-</button>
+      <img class="images" src="./assets/cats-pic/{{cat.picUrl[index]}}" alt="">
+      <button (click) = "scrollRight()" class="scrolls">-></button>
+        <div class="cols">
+            <span class="cats-name">{{cat.name}}</span>
+            <span>{{cat.ageMonths}} miesięcy</span>
+          <div class="vacinTrue">
```

Comment:
całość powinna być ngIf'owana. Inaczej zostaje pusty div, którego tutaj nie widać, ale w innych okolicznościach może zaburzyć layout albo stylowanie. 
Jeśli coś ma zniknąć, to zupełenie: albo wyłączone przez ngIf albo np display:none